api/src/main/java/org/pmiops/workbench/cdrselector/CdrSelectorService.java

```diff
@@ -0,0 +1,11 @@
+package org.pmiops.workbench.cdrselector;
+
+import org.pmiops.workbench.db.model.DbWorkspace;
+import org.pmiops.workbench.model.WorkspaceAccessLevel;
+import org.pmiops.workbench.model.WorkspaceResource;
+
+import java.util.List;
+
```

Comment:
A little comment about what a CDR selector is would help here.

Is `WorkspaceResource` already in use? If not, maybe we could change it (unless a CDR Selector is a subset of workspace resource).

Reply:
A CDR Selector is a subset of workspace resource.